recipes/haslr/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.8a1" %}
+{% set sha256 = "0d55a943a828cd1a42701774b52ef9c2ba1d944a7fc3415d64856ad88c135ee8" %}
+
+package:
+  name: haslr
+  version: {{ version }}
+
+source:
+  url: https://github.com/vpc-ccg/haslr/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # minia is not available for osx
+  skip: True  # [osx or py27]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - zlib
+  host:
+    - python
+    - zlib
+    - minia 3.2.1
+    - minimap2 2.17
+    - fastutils 0.2

```

Comment:
Are these exact version numbers really needed, or should they instead be minimum versions.

Reply:
I think they can be minimum versions. But it seemed the building/testing module was not happy with >= sign. What is the best way to do it? Thanks.

Reply:
Never mind! I updated `meta.yaml` and it seems find with >= sings. Thank you.